crates/macro-support/src/parser.rs

```diff
@@ -618,7 +618,7 @@ impl<'a> ConvertToAst<(&ast::Program, BindgenAttrs, &'a Option<ast::ImportModule
                 ast::ImportFunctionKind::Normal => (0, "n"),
                 ast::ImportFunctionKind::Method { ref class, .. } => (1, &class[..]),
             };
-            let data = (ns, &self.sig.ident, module);
+            let data = (ns, &self.sig, module);
```

Comment:
```suggestion
            let data = (ns, self.sig.to_token_stream().to_string(), module);
```
I know this is a bit messed up ... but its worth it avoiding `extra-traits`.

Reply:
Huh, alright then.